Identiske underkataloger: src2/CVS og src/CVS
Identiske underkataloger: src2/.deps og src/.deps
Index: gaim/src/gtkstatusbox.c
===================================================================
--- gaim.orig/src/gtkstatusbox.c	2005-04-24 21:52:29.000000000 +0200
+++ gaim/src/gtkstatusbox.c	2005-08-19 21:22:49.719366944 +0200
@@ -169,7 +169,7 @@
 	GtkCellRenderer *icon_rend = gtk_cell_renderer_pixbuf_new();
 	GtkTextBuffer *buffer;
 	GdkPixbuf *pixbuf, *pixbuf2, *pixbuf3, *pixbuf4;
-	GtkIconSize icon_size = gtk_icon_size_from_name(GAIM_ICON_SIZE_STATUS);
+	GtkIconSize icon_size = GTK_ICON_SIZE_SMALL_TOOLBAR;
 
 	status_box->imhtml_visible = FALSE;
 	status_box->error_pixbuf = gtk_widget_render_icon (GTK_WIDGET(status_box), GAIM_STOCK_STATUS_OFFLINE,
